src/Enzo/enzo_EnzoComputePressure.cpp

```diff
@@ -106,6 +107,13 @@ void EnzoComputePressure::compute_(Block * block,
     const enzo_float * vy = (enzo_float*) field.values("velocity_y", i_hist_);
     const enzo_float * vz = (enzo_float*) field.values("velocity_z", i_hist_);
 
+    const enzo_float * bx = mhd ?
```

Comment:
Ideally, I think these would use `CelloArrays`, but I was trying to limit how much of the existing code I was changing (to minimize conflicts).

I think it would be better to handle this in the future